dags/utils/slack_operator.py

```diff
@@ -0,0 +1,81 @@
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+
+# This is the Conn Id that we set when creating the connection in the Airflow dashboard
+# in Admin > Connections.
+SLACK_CONN_ID = "slack"
+
+
+def task_fail_slack_alert_critical(context):
+    slack_webhook_token = BaseHook.get_connection(SLACK_CONN_ID).password
+    slack_msg = """
+            <!channel> :red_circle: Critical Failure
```

Comment:
does this mean @ the whole channel? 
I was going to propose a different icon to differentiate from normal failures, but if it alerts the whole channel, thats different enough.

Reply:
Yes, that is exactly what it translates it to in the Slack API. This was 100% copy/paste from the existing Slack operator code, and it is good to know that we have this option if we ever needed that. 🚨

Reply:
whoa! good to know!